imports/plugins/core/graphql/server/schemas/cart.graphql

```diff
@@ -0,0 +1,346 @@
+  price: Float!
+
+  "The currency for the `price` and `priceWhenAdded` fields"
+  priceCurrency: String!
+}
+
+input ProductConfigurationInput {
+  "The Product ID"
+  productId: ID!
+
+  "The ProductVariant ID"
+  productVariantId: ID!
+}
+
+input AddCartItemsInput {
+  "The cart ID"
+  cartId: ID!
+
+  "Array of items to be added to the cart"
+  items: [CartItemInput]!
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+}
+
+input RemoveCartItemsInput {
+  "The cart ID"
+  cartId: ID!,
+
+  "Array of items to remove from the cart."
+  cartItemIds: [ID]!
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+}
+
+input UpdateCartItemInput {
+  "The cart item ID"
+  cartItemId: ID!
+
+  "New absolute value for specified cart item's quantity. Not an incremental value."
+  quantity: Int!
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+}
+
+input UpdateCartItemsQuantityInput {
+  "The cart ID"
+  cartId: ID!
+
+  "Array of cart item quantities to update."
+  items: [UpdateCartItemInput]!
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+}
+
+input ReconcileCartInput {
+  "An anonymous cart ID"
+  anonymousCartId: ID!
+
+  "An anonymous cart token"
+  anonymousCartToken: String!
+
+  "If both an anonymous cart and a cart for the authenticated account are found, how do we combine them? Default mode is `merge`, where all anonymous items are moved into the account cart along with existing account cart items, and quantities are combined."
+  mode: CartReconciliationMode
+
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+}
+
+
+####################
+# Payloads
+# These types are used as return values for mutation calls
+####################
+
+"The payload returned from the `createCart` mutation call"
+type CreateCartPayload {
+  "The modified cart"
+  cart: Cart!
+
+  "The same string you sent with the mutation params, for matching mutation calls with their responses"
+  clientMutationId: String
+}
+
+
+"The payload returned from the `addCartItems` mutation call"
+type AddCartItemsPayload {
+  "The modified cart"
+  cart: Cart!
+
+  "The same string you sent with the mutation params, for matching mutation calls with their responses"
+  clientMutationId: String
+}
+
+
+"The payload returned from the `removeCartItems` mutation call"
+type RemoveCartItemsPayload {
+  "The modified cart"
+  cart: Cart!
+
+  "The same string you sent with the mutation params, for matching mutation calls with their responses"
+  clientMutationId: String
+}
+
+
+"The payload returned from the `updateCartItems` mutation call"
+type UpdateCartItemsPayload {
```

Comment:
Add `Quantity` to match mutation name: `UpdateCartItemsQuantityPayload` (Also change the response down by the mutation definition.)